src/libs/Middleware/SaveResponseInOnyx.js

```diff
@@ -14,11 +14,7 @@ const requestsToIgnoreLastUpdateID = ['OpenApp', 'ReconnectApp', 'GetMissingOnyx
  * @returns {Promise}
  */
 function SaveResponseInOnyx(requestResponse, request) {
-    return requestResponse.then((response) => {
-        // Make sure we have response data (i.e. response isn't a promise being passed down to us by a failed retry request and response undefined)
-        if (!response) {
-            return;
-        }
+    return requestResponse.then((response = {}) => {
         const onyxUpdates = response.onyxData;
```

Comment:
I wonder if there's case of `response = null`, which will make crash on this line.

Reply:
I haven't come across that case yet since I've only seen response = undefined or an actual response object. I could add the null check back if we think it's valuable